lib/rbnacl/group_elements/curve25519.rb

```diff
@@ -61,7 +61,9 @@ def mult(integer)
         Util.check_length(integer, SCALARBYTES, "integer")
 
         result = Util.zeros(SCALARBYTES)
-        self.class.scalarmult_curve25519(result, integer, @point)
+        if self.class.scalarmult_curve25519(result, integer, @point) != 0
+          raise CryptoError "Invalid curve25519 result"
```

Comment:
Missing a comma here, but the test failures seem to indicate this path is being taken in tests that are expected to pass.

This could probably use a test that triggers the offending behavior to ensure the error is raised as expected.